.github/workflows/deploy.yml

```diff
@@ -39,7 +39,9 @@ jobs:
 
       - name: Build package
         run: npm run build
-
+        env:
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
```

Comment:
We should configure the DSN dynamically, sending a new DSN to CLI without the necessity to download the latest version. This is essential for DSN rotation and other maintenance activities. However, we can keep it as is for now.

Reply:
This is a good point. I think we still have time for this, so I'll investigate some effort in finding a solution for this

Reply:
@derevnjuk done.

Please also consider renaming this PR back to `feat`, since the changes introduced by this PR are expected to bump minor version

Reply:
@lsndr, it would be better to release a patch version instead of a minor one. I have updated the release rules accordinalty

Reply:
@lsndr I had to revert the changes in .releaserc. For now, let's create a release manually